src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Copyright(c) Microsoft.All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis.Sarif.Readers;
+using Microsoft.CodeAnalysis.Sarif.VersionOne;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.Visitors
+{
+    public static class SarifTransformerUtilities
```

Comment:
I love how we've hoisted these things twice, to more and more "common" location :-)

Reply:
Time sometimes brings clarity...